validator/remote/src/main/java/tech/pegasys/teku/validator/remote/apiclient/OkHttpValidatorRestApiClient.java

```diff
@@ -250,6 +252,13 @@ public void subscribeToPersistentSubnets(final Set<SubnetSubscription> subnetSub
     post(SUBSCRIBE_TO_PERSISTENT_SUBNETS, subnetSubscriptions, createHandler());
   }
 
+  @Override
+  public Optional<PostSyncCommitteeFailureResponse> sendSyncCommitteeSignatures(
+      final List<SyncCommitteeSignature> syncCommitteeSignatures) {
+    // FIXME
```

Comment:
Probably remove the FIXME comment from here.  We have the ticket still open to follow up on this anyway.

Reply:
this should actually be fairly trivial, i'll make the post.